repair_type/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+No configuration needed for this module.
```

Comment:
Go to Settings > repair types (not sure if this is the path) and configure your repair types.

Reply:
This is already written in USAGE.rst